frontend/public/index.html

```diff
@@ -23,6 +23,10 @@
     <link rel="manifest" href="%PUBLIC_URL%/static/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/static/favicon.ico">
     <title>Kubeflow Pipelines</title>
+    <script>
+      window.KFP_FLAGS={};
+      window.KFP_FLAGS.MULTI_USER=0;
```

Comment:
Does this work for common browsers such as chrome, firefox, safari?

Reply:
Yes, this is very standard html.